pcsx2/gui/Dialogs/McdConfigDialog.cpp

```diff
@@ -155,14 +155,14 @@ void Dialogs::McdConfigDialog::OnMultitapClicked( wxCommandEvent& evt )
 bool Dialogs::McdConfigDialog::Show( bool show )
 {
 	// Suspend the emulation before any file operations on the memory cards can be done.
-	if( show && CoreThread.IsRunning() )
+	if( show && SysHasValidState() )
```

Comment:
So you're not checking anymore that it's running before suspending? Doesn't it break anything?